modules/op-disabling-cluster-tasks-and-pipeline-templates.adoc

```diff
@@ -29,3 +29,7 @@ spec:
         value: 'true'
 ----
 
+[IMPORTANT]
+====
+In {pipelines-title} 1.10, `ClusterTask` functionality  is deprecated and is planned to be removed in a future release.
```

Comment:
```suggestion
In {pipelines-title} 1.10, `ClusterTask` custom resource is deprecated and is planned to be removed in a future release.
```
If it's a CR, why not jut say so? Regardless, there is still an extra space before "is that shows up the previews.

Reply:
Unfortunately it is not just a custom resource, it is the entire way to reference this custom resource (the corresponding YAML section is also named ClusterTask). I really can't make it more specific without a mini-lecture, and just "cluster task" is unfortunately ambiguous in context.